### DISCUSSION TITLE
Problem with jinja2

### DISCUSSION THREAD

**Discussion by palandovalex** [mason-org/mason.nvim#1334] [Q&A]
I am using jinja2. When i open templates (.html extension) neovim do not understand, thai this template is jinja2, it thinks that it is htmldjango.

When i look to code, i dont find specific jinja2 extension. So, what i should do?

**Comment by palandovalex** [comment]
I tried to use .jinja and .jinja2 extensions, but it dosnt understand, what he should use.

**Comment by williamboman** [comment]
This isn't really the place to ask these questions, but you could always:

```vim
autocmd BufRead,BufNewFile *.jinja,*.jinja2 set filetype=jinja2
```

Note that `jinja2` might not be a widely recognized `filetype` in Neovim, so you might run into other issues by setting it to this value.
